src/utils/wallet.js

```diff
@@ -2,7 +2,7 @@ import {starknet} from '../libs';
 import {parseToFelt} from './parser';
 
 export const formatBalance = balance => {
-  return typeof balance === 'number' ? String(fromExponential(balance)) : 'N/A';
+  return balance || 'N/A';
```

Comment:
balance must to be a string? cannot be 0?

Reply:
No, the api now returns a string.